### PR TITLE
SAK-30097 Removed hostname(server-url) from site-info url while 

### PR DESCRIPTION
**Pull request by ouit0408**
storing in DB.

Prepend the server url back to the site-info on the edit screen of home
page.

### CODE REVIEW

File: web/web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java [hunk 1]
```diff
@@ -869,9 +874,13 @@ else if (newPageTitle.length() > MAX_TITLE_LENGTH)
             // Handle the infoUrl
             if (SPECIAL_WORKSITE.equals(special))
             {
-                if ((infoUrl != null) && (infoUrl.length() > 0) && (!infoUrl.startsWith("/")) && (infoUrl.indexOf("://") == -1))
-                {
-                    infoUrl = "http://" + infoUrl;
+                //Check info-url for null and empty
+                if(infoUrl != null && !infoUrl.equals("")){
```
Comment (ern) [inline web/web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java]: Maybe the use of StringUtils.isNotBlank() would be useful here.

File: web/web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java [hunk 2]
```diff
@@ -596,6 +596,11 @@ else if (SPECIAL_WORKSITE.equals(special))
 					    String infoUrl = StringUtils.trimToNull(s.getInfoUrl());
 					    if (infoUrl != null)
 					    {
+                                            //Check if infoUrl is relative? and prepend the server url
+                                            String serverUrl = ServerConfigurationService.getServerUrl();
+                                            if(infoUrl.startsWith("/") && infoUrl.indexOf("://") == -1){
+                                                infoUrl = serverUrl + infoUrl;
```
Comment (ern) [inline web/web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java]: Seems like we would always be calling SCS.getServerUrl even when we don't need to, try something like:
infoUrl = ServerConfigurationService.getServerUrl() + infoUrl;

**Comment by ouit0408** [comment]
yes rightly said, have implemented review comments and pushed the code.

File: web/web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java [hunk 3]
```diff
@@ -596,6 +596,10 @@ else if (SPECIAL_WORKSITE.equals(special))
 					    String infoUrl = StringUtils.trimToNull(s.getInfoUrl());
 					    if (infoUrl != null)
 					    {
+                                            //Check if infoUrl is relative? and prepend the server url
+                                            if(infoUrl.startsWith("/") && infoUrl.indexOf("://") == -1){
```
Comment (ern) [inline web/web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java]: More concise:
if(infoUrl.startsWith("/") && !infoUrl.contains("://"))

**Comment by ouit0408** [comment]
changed it and pushed the code.

**Pull request merged**